src/server.cpp

```diff
@@ -893,6 +893,48 @@ void Server::AsyncRunStep(float dtime, bool initial_step)
 		}
 	}
 
+	/*
+		Send sounds if needed
+	*/
+	{
```

Comment:
Idea: If efficiency is a concern, this could be timed to only be executed once per second -> `IntervalLimiter` instance.

Reply:
I wouldn't prematurely optimize this. If we do want to limit this, the step should be configurable; otherwise modders will be surprised by the seemingly arbitrary timer.